doc/development/theming.rst

```diff
@@ -88,8 +88,8 @@ Python :mod:`ConfigParser` module) and has the following structure:
 Distribute your theme as a Python package
 -----------------------------------------
 
-As a way to distribute your theme, you can use Python package.  Python package
-brings to users easy setting up ways.
+As a way to distribute your theme, you can use a Python package.  This makes it
+easier for users to setup your theme.
```

Comment:
I thought "setup" was the noun and "set up" the verb?

(I am not a maintainer.)

Reply:
Yeah, I think you're right - I've changed it.